pages/components/docs/Flex.md

```diff
@@ -1,41 +1,36 @@
 # Flex and Flex.Item
 
-Flex and Flex.Item are wrapping components that will give the content flexbox properties. Flex.Item is included in the export for Flex.
+Flex is a component that will allow you to use flex box properties
```

Comment:
"Flexbox" should probably be one word. Also, do we want to call them `properties` or `system props`?

```suggestion
Flex is a component that will allow you to use flexbox properties.
```